Procfile

```diff
@@ -1,3 +1,2 @@
 web: bundle exec rackup config.ru --port ${PORT:-3000}
 worker: bundle exec sidekiq --config config/sidekiq.yml
-mail: bundle exec mailcatcher -f
```

Comment:
since `foreman` runs outside of `bundle` I think we can keep `mailcatcher` here in the Procfile? Just remove the `bundle exec` ?